charts/ocis/templates/graph/deployment.yaml

```
@@ -55,6 +55,11 @@ spec:
             - name: GRAPH_SPACES_WEBDAV_BASE
               value: "https://{{ .Values.externalDomain }}"
 
+            {{- if gt (len .Values.features.roles.availableUnifiedRoles) 0 }}
```

Why not just:

```suggestion
            {{ if .Values.features.roles.availableUnifiedRoles }}
```
?

---

right, seems my helm foo is a little bit rusty after my vacation... :laughing: 

Will change it